presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -70,9 +69,6 @@ protected Optional<PlanNodeStatsEstimate> doCalculate(TableScanNode node, StatsP
         return Optional.of(PlanNodeStatsEstimate.builder()
                 .setOutputRowCount(tableStatistics.getRowCount().getValue())
                 .setTotalSize(tableStatistics.getTotalSize().getValue())
-
-                // TODO Handle the confidence level properly when filters are pushed into the tablescan
-                .setConfidence(FACT)
```

Comment:
we do want it to be fact by default when there are no filters, as the stats on just the table without filters are generally reliable, and also the optimizations we run CBO are mostly before filters get pushed into the scan..  Removing this will significantly limit applications that rely on the isConfident() check, and can break exisiting queries.